src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1202,6 +1206,25 @@ begin
   exact (lt_blsub.{u u} (λ x _, x) _ h).false
 end
 
+/-! ### Results about injectivity and surjectivity -/
+
+lemma not_surjective_of_ordinal {α : Type u} (f : α → ordinal.{u}) : ¬ function.surjective f :=
+λ h, ordinal.lsub_nmem_range.{u u} f (h _)
+
+lemma not_injective_of_ordinal {α : Type u} (f : ordinal.{u} → α) : ¬ function.injective f :=
+λ h, not_surjective_of_ordinal _ (inv_fun_surjective h)
+
+lemma not_surjective_of_ordinal_of_small {α : Type v} [small.{u} α] (f : α → ordinal.{u}) :
+  ¬ function.surjective f :=
+λ h, not_surjective_of_ordinal _ (h.comp (equiv_shrink _).symm.surjective)
+
+lemma not_injective_of_ordinal_of_small {α : Type v} [small.{u} α] (f : ordinal.{u} → α) :
+  ¬ function.injective f :=
+λ h, not_injective_of_ordinal _ ((equiv_shrink _).injective.comp h)
+
+theorem not_small_ordinal : ¬ small.{u} ordinal.{max u v} :=
```

Comment:
Please add a docstring that contains the words "Burali-Forti paradox" so that grep/GitHub search can find it.